backend/infrahub/git/base.py

```diff
@@ -195,7 +195,7 @@ def directory_root(self) -> str:
     @property
     def directory_default(self) -> str:
         """Return the path to the directory of the main branch."""
-        return os.path.join(self.directory_root, registry.default_branch)
+        return os.path.join(self.directory_root, "main")
```

Comment:
Hardcode the file system folder to be "main" for the default branch so that the directory structure always looks the regardless of the default branch being used..